desktop-app/app/components/WebView/screenshotUtil.js

```diff
@@ -124,9 +131,11 @@ class WebViewUtils {
     await _delay(200);
   }
 
-  async hideScrollbarAndFixedPositionedElements(): Promise<string> {
+  async hideScrollbarAndFixedPositionedElements(
+    removeFixedPositionedElements: boolean
+  ): Promise<string> {
     const key = await this.webView.insertCSS(`
-      .responsivelyApp__ScreenshotInProgress::-webkit-scrollbar {
+      .body::-webkit-scrollbar {
```

Comment:
Here should be `body` alone. Without the `.`.

Reply:
Good catch, fixed.